src/App.vue

```diff
@@ -504,25 +502,35 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
 
       // ensure NR is valid
       if (!this.isNrValid(nrResponse)) {
+        console.log('NR is not valid') // eslint-disable-line no-console
```

Comment:
I thought this (and the lines below) should log a bit more info on why the NR is invalid.